src/messages.rs

```diff
@@ -40,7 +41,7 @@ pub enum Message {
 /// Messages sent via a direct connection.
 ///
 /// Allows routing to directly send specific messages between nodes.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+#[derive(RustcEncodable, RustcDecodable)]
```

Comment:
This was previously discussed in https://github.com/maidsafe/routing/issues/721 which was closed without a decent resolution in my opinion (I would have preferred something similar to what you're suggesting).